configure.ac

```diff
@@ -476,6 +482,24 @@ AC_ARG_WITH([daemon],
   [build_bitcoind=$withval],
   [build_bitcoind=yes])
 
+if test "x$use_rust" = xyes; then
+  AC_PATH_PROG(RUSTC,rustc)
+  if test x$RUSTC = xnone; then
+    use_rust=no
+    AC_MSG_ERROR("rustc required to enable experimental rust support.")
+  fi
+
+  case $host in
+    ${host_cpu}-w64-mingw*) RUST_TARGET=${host_cpu}-pc-windows-gnu ;;
```

Comment:
Let's perform these checks with `*mingw*`, `*linux*`, and `*darwin*` like we do in other places, or at the very least do the right thing for triples like `aarch64-linux-gnu` (notice that there's no `-pc`)

Reply:
Agree with moving this into the other sections. Want to create a patch for that?

Note that these are already expanded triples, though:
```
 $ ./config.sub aarch64-linux-gnu
aarch64-unknown-linux-gnu
```